crates/cli/src/subcommands/generate/typescript.rs

```diff
@@ -42,15 +41,14 @@ impl Lang for TypeScript {
         reducer_module_name(reducer_name) + ".ts"
     }
 
-    fn generate_type(&self, module: &ModuleDef, namespace: &str, typ: &TypeDef) -> String {
+    fn format_files(&self, _generated_files: BTreeSet<PathBuf>) -> anyhow::Result<()> {
+        // TODO: implement formatting.
```

Comment:
i wonder if there are any formatters written in rust. it would be easier than attempting to automatically interact with npm. Or yarn. or any of the other ones

Reply:
I think it would be fair to assume that any developer working with TypeScript has `npm` installed too (it comes with Node.js after all), so this could be as simple as `npx prettier ...` which auto-installs and invokes it.

I just didn't want to be responsible for choosing the TS formatter, so left the status quo here.